### ISSUE TITLE
Again - Detected blocking call to open with args ('scenes.yaml',)

### ISSUE THREAD

**Issue by Muppetteer** [hugobloem/stateful_scenes#139]
I have seen #105 and #108 but I still see what appears to be the same error logged while running the latest stateful scenes 1.4.0 on home assistant core 2024.11.3 docker:

```
2024-11-27 00:45:07.434 WARNING (MainThread) [homeassistant.util.loop] Detected blocking call to open with args ('scenes.yaml',) inside the event loop by custom integration 'stateful_scenes' at custom_components/stateful_scenes/StatefulScenes.py, line 102: with open(self.scene_path, encoding="utf-8") as f: (offender: /config/custom_components/stateful_scenes/StatefulScenes.py, line 102: with open(self.scene_path, encoding="utf-8") as f:), please create a bug report at https://github.com/hugobloem/stateful_scenes/issues
For developers, please see https://developers.home-assistant.io/docs/asyncio_blocking_operations/#open
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/usr/src/homeassistant/homeassistant/__main__.py", line 223, in <module>
    sys.exit(main())
  File "/usr/src/homeassistant/homeassistant/__main__.py", line 209, in main
    exit_code = runner.run(runtime_conf)
  File "/usr/src/homeassistant/homeassistant/runner.py", line 189, in run
    return loop.run_until_complete(setup_and_run_hass(runtime_config))
  File "/usr/local/lib/python3.12/asyncio/base_events.py", line 674, in run_until_complete
    self.run_forever()
  File "/usr/local/lib/python3.12/asyncio/base_events.py", line 641, in run_forever
    self._run_once()
  File "/usr/local/lib/python3.12/asyncio/base_events.py", line 1990, in _run_once
    handle._run()
  File "/usr/local/lib/python3.12/asyncio/events.py", line 88, in _run
    self._context.run(self._callback, *self._args)
  File "/usr/src/homeassistant/homeassistant/setup.py", line 165, in async_setup_component
    result = await _async_setup_component(hass, domain, config)
  File "/usr/src/homeassistant/homeassistant/setup.py", line 461, in _async_setup_component
    await asyncio.gather(
  File "/usr/src/homeassistant/homeassistant/setup.py", line 463, in <genexpr>
    create_eager_task(
  File "/usr/src/homeassistant/homeassistant/util/async_.py", line 45, in create_eager_task
    return Task(coro, loop=loop, name=name, eager_start=True)
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 783, in async_setup_locked
    await self.async_setup(hass, integration=integration)
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 546, in async_setup
    await self.__async_setup_with_context(hass, integration)
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 635, in __async_setup_with_context
    result = await component.async_setup_entry(hass, self)
  File "/config/custom_components/stateful_scenes/__init__.py", line 32, in async_setup_entry
    hass.data[DOMAIN][entry.entry_id] = Hub(
  File "/config/custom_components/stateful_scenes/StatefulScenes.py", line 78, in __init__
    scene_confs = self.load_scenes()
  File "/config/custom_components/stateful_scenes/StatefulScenes.py", line 102, in load_scenes
    with open(self.scene_path, encoding="utf-8") as f:
```

**Comment by sayam93** [comment]
I get this too. I am running the latest version of the Custom Integration and HA on my HAOS setup.

```
Logger: homeassistant.util.loop
Source: util/loop.py:136
First occurred: 12:48:14 PM (2 occurrences)
Last logged: 12:49:42 PM

Detected blocking call to open with args ('scenes.yaml',) inside the event loop by custom integration 'stateful_scenes' at custom_components/stateful_scenes/StatefulScenes.py, line 102: with open(self.scene_path, encoding="utf-8") as f: (offender: /config/custom_components/stateful_scenes/StatefulScenes.py, line 102: with open(self.scene_path, encoding="utf-8") as f:), please create a bug report at https://github.com/hugobloem/stateful_scenes/issues For developers, please see https://developers.home-assistant.io/docs/asyncio_blocking_operations/#open Traceback (most recent call last): File "<frozen runpy>", line 198, in _run_module_as_main File "<frozen runpy>", line 88, in _run_code File "/usr/src/homeassistant/homeassistant/__main__.py", line 223, in <module> sys.exit(main()) File "/usr/src/homeassistant/homeassistant/__main__.py", line 209, in main exit_code = runner.run(runtime_conf) File "/usr/src/homeassistant/homeassistant/runner.py", line 189, in run return loop.run_until_complete(setup_and_run_hass(runtime_config)) File "/usr/local/lib/python3.12/asyncio/base_events.py", line 674, in run_until_complete self.run_forever() File "/usr/local/lib/python3.12/asyncio/base_events.py", line 641, in run_forever self._run_once() File "/usr/local/lib/python3.12/asyncio/base_events.py", line 1990, in _run_once handle._run() File "/usr/local/lib/python3.12/asyncio/events.py", line 88, in _run self._context.run(self._callback, *self._args) File "/usr/src/homeassistant/homeassistant/setup.py", line 165, in async_setup_component result = await _async_setup_component(hass, domain, config) File "/usr/src/homeassistant/homeassistant/setup.py", line 461, in _async_setup_component await asyncio.gather( File "/usr/src/homeassistant/homeassistant/setup.py", line 463, in <genexpr> create_eager_task( File "/usr/src/homeassistant/homeassistant/util/async_.py", line 45, in create_eager_task return Task(coro, loop=loop, name=name, eager_start=True) File "/usr/src/homeassistant/homeassistant/config_entries.py", line 783, in async_setup_locked await self.async_setup(hass, integration=integration) File "/usr/src/homeassistant/homeassistant/config_entries.py", line 546, in async_setup await self.__async_setup_with_context(hass, integration) File "/usr/src/homeassistant/homeassistant/config_entries.py", line 635, in __async_setup_with_context result = await component.async_setup_entry(hass, self) File "/config/custom_components/stateful_scenes/__init__.py", line 32, in async_setup_entry hass.data[DOMAIN][entry.entry_id] = Hub( File "/config/custom_components/stateful_scenes/StatefulScenes.py", line 78, in __init__ scene_confs = self.load_scenes() File "/config/custom_components/stateful_scenes/StatefulScenes.py", line 102, in load_scenes with open(self.scene_path, encoding="utf-8") as f:
Detected blocking call to open with args ('/usr/local/lib/python3.12/site-packages/pytz/zoneinfo/Asia/Kolkata', 'rb') inside the event loop by custom integration 'trakt_tv' at custom_components/trakt_tv/apis/trakt.py, line 295: pytz.timezone(configuration.get_timezone()) (offender: /usr/local/lib/python3.12/site-packages/pytz/__init__.py, line 108: return open(filename, 'rb')), please create a bug report at https://github.com/dylandoamaral/trakt-integration/issues For developers, please see https://developers.home-assistant.io/docs/asyncio_blocking_operations/#open Traceback (most recent call last): File "<frozen runpy>", line 198, in _run_module_as_main File "<frozen runpy>", line 88, in _run_code File "/usr/src/homeassistant/homeassistant/__main__.py", line 223, in <module> sys.exit(main()) File "/usr/src/homeassistant/homeassistant/__main__.py", line 209, in main exit_code = runner.run(runtime_conf) File "/usr/src/homeassistant/homeassistant/runner.py", line 189, in run return loop.run_until_complete(setup_and_run_hass(runtime_config)) File "/usr/local/lib/python3.12/asyncio/base_events.py", line 674, in run_until_complete self.run_forever() File "/usr/local/lib/python3.12/asyncio/base_events.py", line 641, in run_forever self._run_once() File "/usr/local/lib/python3.12/asyncio/base_events.py", line 1990, in _run_once handle._run() File "/usr/local/lib/python3.12/asyncio/events.py", line 88, in _run self._context.run(self._callback, *self._args) File "/config/custom_components/trakt_tv/apis/trakt.py", line 316, in fetch_next_to_watch data = await self.fetch_upcoming( File "/config/custom_components/trakt_tv/apis/trakt.py", line 295, in fetch_upcoming pytz.timezone(configuration.get_timezone())
```

**Comment by hugobloem** [comment]
Should be fixed with #141
